src/services/services.ts

```diff
@@ -391,9 +453,25 @@ namespace ts {
                     // If it is parameter - try and get the jsDoc comment with @param tag from function declaration's jsDoc comments
                     if (canUseParsedParamTagComments && declaration.kind === SyntaxKind.Parameter) {
                         ts.forEach(getJsDocCommentTextRange(declaration.parent, sourceFileOfDeclaration), jsDocCommentTextRange => {
-                            const cleanedParamJsDocComment = getCleanedParamJsDocComment(jsDocCommentTextRange.pos, jsDocCommentTextRange.end, sourceFileOfDeclaration);
-                            if (cleanedParamJsDocComment) {
-                                addRange(jsDocCommentParts, cleanedParamJsDocComment);
+                            // Incoming function strips off leading /* and trailing */, so add those back (-2, +4 net)

```

Comment:
"Incoming function"? Do you mean `parseIsolatedJSDocComment`? If so, make that explicit.